pkg/controller/cluster/agent/agent.go

```diff
@@ -48,7 +48,7 @@ func ensureObject(ctx context.Context, cfg *Config, obj ctrlruntimeclient.Object
 
 	if result != controllerutil.OperationResultNone {
 		key := client.ObjectKeyFromObject(obj)
-		log.Info(fmt.Sprintf("ensuring %T", obj), "key", key, "result, result")
+		log.Info(fmt.Sprintf("ensuring %T", obj), "key", key, "result", result)
```

Comment:
can we use `WithValues` instead of sprintf just to be consistent

Reply:
`WithValues` is not the same though, it returns a new logger with the key-value pairs.

The Sprintf is just used to format the first argument (the msg): `Info(msg string, keysAndValues ...any)` :slightly_smiling_face: